services/github/src/utils.js

```diff
@@ -59,18 +61,37 @@ throttleRequest.request = request;
  * @returns boolean
  */
 const shouldSkipCommit = ({ commits, head_commit = {} }) => {
-  const testRe = new RegExp('\\[(skip ci|ci skip)\\]', 'i');
-
   let last_commit = head_commit && head_commit.message ? head_commit : false;
 
   if (!last_commit && Array.isArray(commits) && commits.length > 0) {
     last_commit = commits[commits.length - 1];
   }
 
-  return last_commit && testRe.test(last_commit.message);
+  return last_commit && ciSkipRegexp.test(last_commit.message);
+};
+
+module.exports = {
+  throttleRequest,
+  shouldSkipCommit,
+};
+
```

Comment:
Duplicate to lines 93-97? https://github.com/taskcluster/taskcluster/pull/5617/files#diff-8f483f322da8f489a711c40bdb0698a1c741703dcb50232815073986726a2ecfR93-R97

```suggestion
```